packages/runtime/src/components/chakra-ui/Image.tsx

```diff
@@ -123,7 +123,17 @@ export default {
     version: 'chakra_ui/v1',
     metadata: {
       name: 'image',
+      displayName: 'Image',
       description: 'chakra_ui image',
+      isDraggable: true,
+      isResizable: true,
+      defaultProperties: {
```

Comment:
Since this kind of default value looks like a set of subjective mock values and is mostly used in editor mode(I think?), is it possible to make the code/naming more concrete?

Like `exampleProperties or `editorInitProperties` or other smart namings. Because default properties sound like the best practice props in runtime and end-users can use the component really well without other props.

Reply:
Good point. exampleProperties will be better.